docs/docs/features/underglow.md

```diff
@@ -34,41 +34,47 @@ CONFIG_ZMK_RGB_UNDERGLOW=y
 CONFIG_WS2812_STRIP=y
 ```
 
-See [Configuration Overview](/docs/config) for more instructions on how to
-use Kconfig.
+See [Configuration Overview](/docs/config) for more instructions on how to use Kconfig.
 
 If your board or shield does not have RGB underglow configured, refer to [Adding RGB Underglow to a Board](#adding-rgb-underglow-to-a-board).
 
 ### Modifying the Number of LEDs
 
-A common issue when enabling underglow is that some of the installed LEDs do not illuminate. This can happen when a board's default underglow configuration accounts only for either the downward facing LEDs or the upward facing LEDs under each key. On a split keyboard, a good sign that this may be the problem is that the unilluminated LEDs on each half are symmetrical.
+If some of the LEDs on your keyboard don't light up, that could be a hardware problem. But another possibility is that the number of LEDs in the default configuration doesn't match the number that are installed. For example, the `corne` shield specifies only 10 LEDs per side while you might have as many as 27. On a split keyboard, a good sign of this problem is if the lit LEDs on each half are symmetrical.
 
-The number of underglow LEDs is controlled by the `chain-length` property in the `led_strip` node. You can [change the value of this property](../config/index.md#changing-devicetree-properties) in the `<keyboard>.keymap` file by adding a stanza like this one outside of any other node (i.e. above or below the `/` node):
+The `chain-length` property of the `led_strip` node controls the number of underglow LEDs. If it is incorrect for your build, [you can change this property](../config/index.md#changing-devicetree-properties) in your `<keyboard>.keymap` file by adding a stanza like this one outside of any other node (i.e. above or below the `/` node):
 
 ```dts
 &led_strip {
     chain-length = <21>;
 };
 ```
 
-where the value is the total count of LEDs (per half, for split keyboards).
+For split keyboards, set `chain-length` to the number of LEDs installed on each half.
```

Comment:
I'm not entirely happy with the subsection name considering the text. It makes sense to make a note of this somewhere for cases like the one described, but the name doesn't seem "troubleshooting" enough for my taste. It may also make sense to move this out into a "feature troubleshooting" section at some point.

Reply:
Maybe change it to something like "Troubleshooting Partially Working LEDs" (a mouthful, so maybe not the best) for now?